cypress/integration/Sidebar.spec.ts

```diff
@@ -102,7 +102,9 @@ describe('<SideBar />', () => {
 
         cy.get('[data-cy="autocomplete-input"').should('have.value', '');
         const listedCountries = cy.get('[data-cy="listed-country"]');
-        listedCountries.should('have.length.gte', 5);
+        listedCountries.should('have.length.gte', 4);
```

Comment:
Why this number was changed from 5 to 4? In the fixture there are 5 countries that are being returned as mock response

Reply:
I think it was an old commit (done for testing purposes on failing test), it should be back 5. Please double check.

Reply:
All good now 👍